### PR TITLE
Add mozcn safebrowsing for release variant

### PR DESCRIPTION
**Pull request by rxumoz**
Add mozcn safebrowsing provider for release variant since #16384 has just done the beta & nightly part.

### CODE REVIEW

**Review by Amejia481 — APPROVED** [review]
LGTM!

File: app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt [hunk 1]
```diff
@@ -15,10 +15,15 @@ import org.mozilla.fenix.Config
...
 import org.mozilla.geckoview.GeckoRuntimeSettings
+import org.mozilla.geckoview.ContentBlocking.SafeBrowsingProvider
 
 object GeckoProvider {
     var testConfig: Bundle? = null
     private var runtime: GeckoRuntime? = null
+    const val CN_UPDATE_URL =
+        "https://sb.firefox.com.cn/downloads?client=SAFEBROWSING_ID&appver=%MAJOR_VERSION%&pver=2.2"
+    const val CN_GET_HASH_URL =
+        "https://sb.firefox.com.cn/gethash?client=SAFEBROWSING_ID&appver=%MAJOR_VERSION%&pver=2.2"
```
Comment (Amejia481) [inline app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt:26]: I tried to do some HTTP GET to confirm if the URLs are correct, but I got 405 verylike that GV add some tokens to their requests.

Comment (l-hedgehog) [reply app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt:26]: Hi @Amejia481, both endpoints only accept POST requests, as described in https://code.google.com/archive/p/google-safe-browsing/wikis/Protocolv2Spec.wiki

```bash
l-hedgehog@github MINGW64 ~
$ curl -d "m6eb-phish-shavar;a:1-2688:s:1-8" -H "Content-Type: text/plain" "https://sb.firefox.com.cn/downloads?pver=2.2"
n:43200
i:m6eb-phish-shavar
u:sbc.firefox.com.cn/3/L5/FSFXOEBDCOGUVFS7XLGE476OXUAMQYG4UWPY7CGDAVAPEQS3XA
```

**Review by pocmo — APPROVED** [review]

File: app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt (hunk 1 above)
Comment (Amejia481) [reply app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt:26]: Thanks for the clarification :)

**Pull request merged**